xcode/Subconscious/SubconsciousTests/Tests_DatabaseService.swift

```diff
@@ -104,12 +104,13 @@ class Tests_DatabaseService: XCTestCase {
             body: "Foo"
```

Comment:
Stray thought: do we have any test cases asserting behaviour for hidden slugs (i.e. `_` prefixed slugs)?

Reply:
Added a couple of quick tests.